### PR TITLE
Post: why content-centric platforms depend heavily on caching

### PR DESCRIPTION
**Pull request by malaskowski**

### CODE REVIEW

File: content/blog/2023/why-content-centric-architectures-depend-on-caching/index.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+An example of a content-centric architecture is a platform built around the Content Management System (CMS). In such a case, the CMS is the single source of truth of all integrations work through it.
+
+While this approach has advantages, it also has some drawbacks. In this blog, we will focus on one of them - dealing with a high volume of changing data like product information.
+
+## Content-centric caching
+
+In the case of content-centric applications, caching usually happens between the CMS and the end user, and it is usually a read-through cache with a time-to-live (TTL) invalidation strategy.
+
+For example, in the Adobe Experience Manager (AEM), the caching happens on the Dispatcher level (Apache Server module). The Dispatcher proxies the request to AEM, caches the responses, and serves them to the end users.
```
Comment (tomaszmichalak) [inline content/blog/2023/why-content-centric-architectures-depend-on-caching/index.md:24]:
```suggestion
For example, in the Adobe Experience Manager (AEM), the caching happens on the Dispatcher level (Apache Server module). The Dispatcher proxies all the requests to AEM Publish instances, caches the responses, and serves them to the end users.
```

Comment (tomaszmichalak) [inline content/blog/2023/why-content-centric-architectures-depend-on-caching/index.md:24]: Maybe you should mention that the hardest part of Dispatcher caching are invalidation rules. They depend on complex content relations (PDP pages and PLP pages etc) and .stat files. More here: https://experienceleague.adobe.com/docs/experience-manager-dispatcher/using/configuring/dispatcher-configuration.html?lang=en

When we refer to AEM, we must be very precise.

File: content/blog/2023/why-content-centric-architectures-depend-on-caching/index.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+![AEM caching](aem-caching.png)
+
+## Sample solutions
+
+To make things more clear, let's work with some examples. We will use a simple Product Details Page reference. The solution involves the CMS, which is the source of content and layouts of pages. Product Information Management (PIM) system is the source of product data presented on the product pages.
```
Comment (tomaszmichalak) [inline content/blog/2023/why-content-centric-architectures-depend-on-caching/index.md:30]:
```suggestion
To make things more clear, let's work with some example. We will use a simple Product Details Page reference. The solution involves the CMS, which is the source of content and layouts of pages. Product Information Management (PIM) system is the source of product data presented on the product pages.
```

**Review by tomaszmichalak — CHANGES_REQUESTED** [review]

File: content/blog/2023/why-content-centric-architectures-depend-on-caching/index.md (hunk 1 above)
Comment (malaskowski) [reply content/blog/2023/why-content-centric-architectures-depend-on-caching/index.md:24]: I will stick to the previous version.

**Pull request merged**